multisearch_field/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Field MultiSearch with separator",
+    "license": "AGPL-3",
+    "summary": "add modification on search to search multi_value  with separator",
+    "version": "16.0.1.0.0",
+    "author": "Akretion, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/server-ux",
+    "depends": [
+        "base",
+    ],
+    "application": False,
```

Comment:
Not mandatory but it seems this key is useless whereas `maintainers` key can be useful